main.sh

```diff
@@ -109,9 +109,16 @@ if [ $fg_only == 'false' ]; then
        fi
     fi
 fi
-
+ 
 if [ $do_snowDA == 'true' ]; then
-  if [ $hr == '00' ]; then 
+ if [ $hr == '00' ]; then # only calling land DA at 00 
+
+    # if this is a warm start, skip the land DA   
```

Comment:
do you mean that fg_only is true so only the first-guess forecasts are run (so the DA is already done?).  To me, 'warm start' means using the FV3 restart files instead of the GFS 'cold start' files created by chgres_cube

Reply:
Changed to: 
'# check if land DA has already been done." 

'warm start' was a bad choice of words. The situation this check prevents is that if the replay is paused and the land DA has already been done, and the replay is then restarted the land DA would be repeated.